include/asterisk/channel.h

```diff
@@ -4201,6 +4201,8 @@ int ast_channel_hold_state(const struct ast_channel *chan);
 void ast_channel_hold_state_set(struct ast_channel *chan, int value);
 int ast_channel_vstreamid(const struct ast_channel *chan);
 void ast_channel_vstreamid_set(struct ast_channel *chan, int value);
+int ast_channel_tstreamid(const struct ast_channel *chan);
```

Comment:
The *stream functions here are for file playback/recording. This isn't supported for text, so there's no need to have these.

Reply:
Will be removed